WikiDrips/WikiRequest.swift

```diff
@@ -17,9 +17,9 @@ public class WikiRequest
         self.searchText = searchText
     }
     
-    public func fetchWikiDocs(handler: @escaping ([WikiDoc]) -> Void) {
+    public func fetchWikiDocs(handler: @escaping ([WikiDoc]) -> Void) -> URLSessionDataTask? {
```

Comment:
If you only intend to call this `handler` on success, I think it would help to call it something like `successHandler` so that whoever calls this method will know their block won't be called on failure. In our LSM app, and the pattern that Apple usually follows, we usually call completion blocks always and pass out an error through the completion block parameter.